spec/requests/api/targets/create_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe 'POST /api/v1/targets', type: :request do
+  let(:title) { Faker::Lorem.sentence }
+  let(:radius) { Faker::Number.between(from: 50, to: 500) }
+  let(:latitude) { Faker::Address.latitude }
+  let(:longitude) { Faker::Address.longitude }
+  let(:user) { create(:user) }
+  let(:topic) { create(:topic) }
+  let(:target) { Target.last }
+
+  let(:params) do
+    {
+      target: {
+        title: title,
+        radius: radius,
+        latitude: latitude,
+        longitude: longitude,
+        topic_id: topic.id
+      }
+    }
+  end
+
+  subject(:create_target) do
+    post api_v1_targets_path, params: params, headers: auth_headers, as: :json
+  end
+
+  context 'using valid authentication data' do
```

Comment:
what do you think about adding a test about trying to create an invalid target (with invalid data)?